test/unit/dinero.spec.js

```diff
@@ -239,16 +239,22 @@ describe('Dinero', () => {
   describe('#allocate', () => {
     test('should allocate the amount of the Dinero object into new ones and distribute the remainder when given percentages', () => {
       const shares = Dinero({ amount: 1003 }).allocate([50, 50])
-      expect(shares[0].getAmount()).toBe(502)
-      expect(shares[1].getAmount()).toBe(501)
+      expect(shares.map(share => share.getAmount())).toEqual([502, 501])
     })
     test('should allocate the amount of the Dinero object into new ones and distribute the remainder when given ratios', () => {
       const shares = Dinero({ amount: 100 }).allocate([1, 3])
-      expect(shares[0].getAmount()).toBe(25)
-      expect(shares[1].getAmount()).toBe(75)
+      expect(shares.map(share => share.getAmount())).toEqual([25, 75])
     })
-    test('should throw when one of the ratios equals to zero', () => {
-      expect(() => Dinero({ amount: 1003 }).allocate([60, 0, 10, 30])).toThrow()
+    test('should allocate the amount of the Dinero object into new ones and distribute the remainder when one of the ratios equals to zero', () => {
+      const shares = Dinero({ amount: 1003 }).allocate([0, 50, 50])
+      expect(shares.map(share => share.getAmount())).toEqual([0, 502, 501])
+    })
+    test('should allocate the small amount of the Dinero object into new ones and distribute the remainder when one of the ratios equals to zero', () => {
```

Comment:
It's a bit unclear to me what this test demonstrates. Can you please clarify the description?

Reply:
Sorry, I should have described the problem more clearly.

I deleted the test. Probably we should talk about it in another issue.

The test shows a problem with the distribution of the remainder between the ratios `[1, 99]`. The current version returns the distribution of `[1, 0]`. Probably next major version should returns the distribution of `[0, 1]`.

Reply:
No worries. I addressed this point [here](https://github.com/dinerojs/dinero.js/pull/85#pullrequestreview-371924185) but indeed, feel free to open a new issue so we can elaborate more on it.